### ISSUE TITLE
Add the tools to brew

### ISSUE THREAD

**Issue by szaitsev** [fosslife/devtools-x#63]
Brew is a standard way for many people to install tools on Mac. It would be good to have the ability to install devtools-x via brew.

**Comment by Sparkenstein** [comment]
That's a nice idea, I will look into it. PRs are appreaciated if anyone knows the process

**Comment by yemikudaisi** [comment]
While we are on this, it would make sense to also consider `winget` (Windows), `flatpak` and `snap` (Linux).

**Comment by Hacksore** [comment]
The big lore/requirements are here and I've successfully got another [tauri app on homebrew](https://github.com/Homebrew/homebrew-cask/pull/167332) before.

https://docs.brew.sh/Acceptable-Casks

**Comment by Hacksore** [comment]
got a draft pr up here 🙏
https://github.com/Homebrew/homebrew-cask/pull/183913

**Comment by Hacksore** [comment]
We have a big blocker here, the macOS version needs to be signed...

```
audit for devtools-x: failed
 - Signature verification failed:
/private/tmp/cask-audit20240830-4524-8z3xc5/dev-tools.app: code has no resources but signature indicates they must be present

macOS on ARM requires software to be signed.
Please contact the upstream developer to let them know they should sign and notarize their software.
```

**Comment by Hacksore** [comment]
Fixed the syntax CI check but if the apps not signed it can't go on homebrew for ARM 😞.

> App fails with GateKeeper enabled on Homebrew supported macOS versions and platforms (e.g. unsigned apps fail on Macs with Apple Silicon/ARM).

I could try to omit the ARM build until it's signed, let me try that.

**Comment by Hacksore** [comment]
Yeah signing is required as per https://github.com/Homebrew/homebrew-cask/pull/183913#issuecomment-2322505849

Tag me if anything changes here and I'll resist this.
